samples/otlptracegrpc/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "sample-app-grpc",
+  "version": "0.1.0",
+  "private": true,
+  "author": "Google Inc.",
+  "license": "Apache-2.0",
+  "scripts": {
+    "start": "ts-node ./app.ts",
+    "client": "ts-node ./client.ts",
+    "lint": "gts lint",
+    "clean": "gts clean",
+    "fix": "gts fix",
+    "compile": "tsc"
+  },
+  "keywords": [],
+  "dependencies": {
+    "@opentelemetry/api": "1.8.0",
+    "@opentelemetry/exporter-trace-otlp-http": "0.49.1",
+    "@opentelemetry/resources": "1.22.0",
+    "@opentelemetry/sdk-node": "0.49.1",
+    "@opentelemetry/semantic-conventions": "1.22.0",
+    "@opentelemetry/exporter-trace-otlp-grpc": "^0.51.0",
+    "@grpc/grpc-js": "^1.3.2",
+    "express": "4.19.2",
+    "google-auth-library": "9.7.0",
+    "axios": "^1.6.8"
+  },
+  "devDependencies": {
+    "gts": "^5.3.0"
```

We should add typescript to not depend on GTS's transitive dep
```suggestion
    "gts": "5.3.0",
    "typescript": "4.9.5"
```